pages/contact.vue

```diff
@@ -49,20 +29,12 @@
             </i18n>
           </v-card-text>
         </v-card>
-        <h4>
-          <span class="red--text">*</span>
-          {{ $t('contact.subject') }}
-          <span class="red--text">({{ $t('contact.required') }})</span>
-        </h4>
-        <v-text-field v-model="selectedSubject" :label="$t('contact.subject')" solo />
-        <h4>
-          <span class="red--text">*</span>
-          {{ $t('contact.message') }}
-          <span class="red--text">({{ $t('contact.required') }})</span>
-        </h4>
-        <v-textarea v-model="selectedMessage" :label="$t('contact.message')" solo />
         <div id="other-contacts">
           <h2>{{ $t('contact.other-channels') }}</h2>
+          <h4>{{ $t('contact.by-email') }}</h4>
+          <a href="mailto:email.address@canada.ca">
```

Comment:
`email.address@canada.ca` should be `woudcEmail`. As well as changing `href=` to `:href=` for data/variable binding.